CMakeLists.txt

```diff
@@ -255,6 +261,15 @@ if(CCACHE_FOUND)
   set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
 endif(CCACHE_FOUND)
 
+set(SENTRY_BACKEND "crashpad")
+set(SENTRY_INTEGRATION_QT "ON")
+set(SENTRY_INCLUDE_DIR "3rdparty/sentry-native/include")
+set(SENTRY_LIBRARIES "3rdparty/sentry-native")
+
+if(APPLE OR WIN32)
+  add_subdirectory(3rdparty/sentry-native)
```

Comment:
Like the MacOS PR these all need to be conditional on something like a USE_SENTRY CMake variable.